### PR TITLE
Feature/more UI coverage2

### PR DESCRIPTION
**Pull request by khdegraaf**
test coverage

### CODE REVIEW

**Comment by khdegraaf** [comment]
The code coverage isn't really down.  The configuration changes should maybe help with more consistent stats and not losing UI coverage.  Can't tell until it is merged in though for sure.

File: .github/workflows/go.yml
Comment (ryardley) [inline .github/workflows/go.yml:32]: Not sure I understand what this change does and why make install is better.

File: .github/workflows/node.yml
Comment (ryardley) [inline .github/workflows/node.yml:31]: Does this reuse the preexisting binary that is cached?

**Review by ryardley — COMMENTED** [review]
I would like to review this but I am not sure exactly what it is doing so hopefully you don't mind I ask some potentially silly or basic questions?

File: .github/workflows/node.yml
Comment (khdegraaf) [reply .github/workflows/node.yml:31]: No, yarn chain:build ends up do a `make build`, so this is unnecessary and broken.

File: .github/workflows/go.yml
Comment (khdegraaf) [reply .github/workflows/go.yml:32]: the go build command literally doesn't work.  Make install has lots of detailed options and works.

File: .github/workflows/node.yml
Comment (ryardley) [reply .github/workflows/node.yml:31]: Ahh that's right it does too.

**Review by 59023g — CHANGES_REQUESTED** [review]
I'd like to hold off on this until at least after the release. There have been blocking inconsistencies with the existing code coverage action, so I'm hesitant to change it more. Especially now, and without further dialogue amongst stakeholders (cc @ryardley).

**Comment by khdegraaf** [comment]
This change might fix those inconsistencies.  I don't believe code coverage can block merges either.  It is an optional check, not a required one.  I suspect this failed temporarily, but you were blocked due to something else.

File: codecov.yml
Comment (utx0) [inline codecov.yml:50]: whats having these files in here do? They arent ignored I hope?

**Comment by khdegraaf** [comment]
Merged elsewhere.

**Pull request closed without merging**
